applications/asset_tracker_v2/src/cloud/cloud_codec/json_common.c

```diff
@@ -12,10 +12,13 @@
 #include "json_common.h"
 #include "json_helpers.h"
 #include "json_protocol_names.h"
+#include <net/nrf_cloud_location.h>
 
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(json_common, CONFIG_CLOUD_CODEC_LOG_LEVEL);
 
+#define AP_STRING_SIZE	sizeof("deadbeef1122")
```

Comment:
```suggestion
/* MAC address string size including null character */
#define AP_STRING_SIZE	13
```

Reply:
Why not just write the size directly? This way the user don't need to count the characters if their debugging this code for some reason and want to know.